### ISSUE TITLE
SasView Menu bar clarity (Trac #515)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#648]
Two things to fix in the top menubar:

* Tool should be changed to Tools
* The Analysis item should be moved to be next to the Analysis item that is active (currently only Fitting has such a menu)

Migrated from http://trac.sasview.org/ticket/515
```json
{
    "status": "closed",
    "changetime": "2016-03-17T09:17:02",
    "_ts": "2016-03-17 09:17:02.316899+00:00",
    "description": "Two things to fix in the top menubar:\n\n* Tool should be changed to Tools\n* The Analysis item should be moved to be next to the Analysis item that is active (currently only Fitting has such a menu)\n",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-03-15T17:20:13",
    "component": "SasView",
    "summary": "SasView Menu bar clarity",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.0.0",
    "owner": "",
    "type": "enhancement"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/03/16 22:24:00`: 
* **butler** commented:

Forgot to add "closes #648" to commit message so closing ticket manually.  Steve changed the first part and Paul fixed the menu order during Code Camp IV.


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"
* **butler** changed type from "defect" to "enhancement"

**Comment by butlerpd** [comment]
Trac update at `2016/03/17 09:17:02`: **butler** changed summary from "!SasView Menu bar clarity" to "SasView Menu bar clarity"
